benchmarks/urls.py

```diff
@@ -12,13 +14,13 @@
     path('/', user.LandingPage.as_view(), name='landing_page'),
 
     # global pages
```

Comment:
Should we add explore as a global page similar to compare? `Explore` link breaks if navigating from any non-leaderboard page as /{domain}/ is not in the url path?

Reply:
How do we handle this for the leaderboard? I guess we can just do the same here

Reply:
When clicking leaderboard from Tutorial or FAQ, it takes you make to the home page where you then need to click vision/language. If we modify explore to global page, we can default the explore page to vision/explore but if the user is on the language leaderboard and clicks explore, it will take to language/explore.

This is the same logic that is used for Compare.